packages/playground/blueprints/src/lib/steps/run-sql-file.ts

```diff
@@ -0,0 +1,48 @@
+import { StepHandler } from '.';
+
+/**
+ * @inheritDoc runSqlFile
+ * @hasRunnableExample
+ * @example
+ *
+ * <code>
+ * {
+ * 	    "step": "runSqlFile",
+ * 		"path": "wordpress.org/plugins"
+ * }
+ * </code>
+ */
+export interface RunSqlFileStep {
+	/**
+	 * The step identifier.
+	 */
+	step: 'runSqlFile';
+	/**
+	 * The SQL file to run.
+	 */
+	path: string;
+}
+
+/**
+ * Run an SQL file.
+ *
+ * @param playground The playground client.
+ * @param path The path to the SQL file.
+ */
+export const runSqlFile: StepHandler<RunSqlFileStep> = async (
+	playground,
+	{ path },
+	progress?
+) => {
+	progress?.tracker.setCaption(`Executing SQL File`);
+
+	const runPhp = await playground.run({
+		code: `<?php
+		require_once '/wordpress/wp-load.php';
+		require_once '/wordpress//wp-content/plugins/Collector/Collector_Restore.php';
```

Comment:
This creates an implicit dependency on the collector plugin which may not be installed when this step runs. SQL restoring logic only seems relevant in the Playground context so maybe let's ship that PHP file right in this PR? Then you could do this:

```ts

/** @ts-ignore */
import collectorRestore from './Collector_Restore.php?raw';

// ...

	await playground.writeFile('/tmp/collector_restore.php', collectorRestore);
	const runPhp = await playground.run({
		code: `<?php
		require_once '/wordpress/wp-load.php';
		require_once '/tmp/collector_restore.php';
	`
	});
```

Reply:
@adamziel This file has been removed.

https://github.com/WordPress/wordpress-playground/pull/745/files